test/e2e/tests/phishing-controller/phishing-detection.spec.ts

```diff
@@ -105,6 +110,9 @@ describe('Phishing Detection', function (this: Suite) {
     };
 
     it('should redirect users to the the MetaMask Phishing Detection page when an iframe domain is on the phishing blocklist', async function () {
+      if (process.env.SELENIUM_BROWSER === 'firefox') {
+        this.skip();
+      }
```

Comment:
Note to facilitate future stable-sync PR creation: this addition is not part of the [original PR](https://github.com/MetaMask/metamask-extension/pull/36634), but was added in a [follow-up PR](https://github.com/MetaMask/metamask-extension/pull/36858) required to fix some flaky tests.